### ISSUE TITLE
Multiple errors 

### ISSUE THREAD

**Issue by aoprea1982** [tatanus/SPF#20]
Got this errors.
Also attaching a screenshot.

########
 File "/root/SPF/spf/core/../web.py", line 281, in start
    cert_path = m.group(1)
AttributeError: 'NoneType' object has no attribute 'group'
########

########
[VERBOSE] /bin/bash: certonly: command not found
########

![spf12](https://user-images.githubusercontent.com/16059481/30733126-95ce3014-9fb0-11e7-8dda-da6fa3e2ce6e.jpg)

// Operating system used to compile and run SPF.
Kali Linux 2017 
Ubuntu 16.04.3 LTS

// Shell command trying to run 
./spf.py -d example.com -g -v -v -f  targets.txt  --simulate -w

![spf15](https://user-images.githubusercontent.com/16059481/30733426-d1ba2d3e-9fb1-11e7-8a7e-5cda01a7db5f.jpg)

**Comment by tatanus** [comment]
Yeah... I am aware of the fact that I pushed a slightly broken version of SPF to GIT.

As soon as I get back in town I will address the issues and update.

Sorry for the inconveniences.

**Comment by tatanus** [comment]
There was an issue with the way I was handling SSL certs. Unfortunately I had to remove the ssl cert support at the moment, but everything else should be working again.
